surfer/viz.py

```diff
@@ -1341,7 +1352,9 @@ def add_label(self, label, color=None, alpha=1, scalar_thresh=None,
             label directory rather than in the label directory itself (e.g.
             for ``$SUBJECTS_DIR/$SUBJECT/label/aparc/lh.cuneus.label``
             ``brain.add_label('cuneus', subdir='aparc')``).
-
+        **kwargs: additional keyword arguments
+            These are passed to the underlying
+            `mayavi.mlab.pipeline.surface` call.
```

Comment:
Can you check all the renderings on CircleCI for errors? For example this one I know will break because you got rid of the section newline -- and indeed the `Notes` section heading is broken:

https://293-1551431-gh.circle-artifacts.com/0/html/generated/surfer.Brain.html#surfer.Brain.add_label

Reply:
Sorry for not very familiar with CircleCI and HTML rendering rules. But actually, the error is in the master branch, see [https://pysurfer.github.io/generated/surfer.Brain.html#surfer.Brain.add_label](https://pysurfer.github.io/generated/surfer.Brain.html#surfer.Brain.add_label). I can try to fix it, but it is not because of my code.

Reply:
See also this [link](https://github.com/sphinx-doc/sphinx/issues/2304).

Reply:
Finally, I figured out how to force line breaking using sphinx...

Reply:
Oh, I thought you meant that the code outside the margin was broken.